monotime/example_test.go

```diff
@@ -0,0 +1,14 @@
+package monotime_test
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aristanetworks/goarista/monotime"
+)
+
+func Example() {
+	start := monotime.Now()
+	time.Sleep(1 * time.Nanosecond)
+	fmt.Println(monotime.Since(start))
```

Comment:
Along the lines of the previous comment, this could sleep a random time (maybe between 1 and 100 ns or so) and verify the time after the call